test/dummy.go

```diff
@@ -12,34 +12,73 @@ import (
 	"github.com/rollkit/go-da"
 )
 
+// The following consts are copied from appconsts to avoid dependency hell
+const (
+	// NamespaceVersionSize is the size of a namespace version in bytes.
+	NamespaceVersionSize = 1
+
+	// NamespaceIDSize is the size of a namespace ID in bytes.
+	NamespaceIDSize = 28
+
+	// NamespaceSize is the size of a namespace (version + ID) in bytes.
+	NamespaceSize = NamespaceVersionSize + NamespaceIDSize
+
+	// ShareSize is the size of a share in bytes.
+	ShareSize = 512
+
+	// ShareInfoBytes is the number of bytes reserved for information. The info
+	// byte contains the share version and a sequence start idicator.
+	ShareInfoBytes = 1
+
+	// ContinuationSparseShareContentSize is the number of bytes usable for data
+	// in a continuation sparse share of a sequence.
+	ContinuationSparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
+
+	// DefaultGovMaxSquareSize is the default value for the governance modifiable
+	// max square size.
+	DefaultGovMaxSquareSize = 64
+
+	DefaultMaxBytes = DefaultGovMaxSquareSize * DefaultGovMaxSquareSize * ContinuationSparseShareContentSize
+)
+
 // DummyDA is a simple implementation of in-memory DA. Not production ready! Intended only for testing!
 //
 // Data is stored in a map, where key is a serialized sequence number. This key is returned as ID.
 // Commitments are simply hashes, and proofs are ED25519 signatures.
 type DummyDA struct {
-	mu      *sync.Mutex // protects data and height
-	data    map[uint64][]kvp
-	height  uint64
-	privKey ed25519.PrivateKey
-	pubKey  ed25519.PublicKey
+	mu          *sync.Mutex // protects data and height
+	data        map[uint64][]kvp
+	maxBlobSize uint64
+	height      uint64
+	privKey     ed25519.PrivateKey
+	pubKey      ed25519.PublicKey
 }
 
 type kvp struct {
 	key, value []byte
 }
 
 // NewDummyDA create new instance of DummyDA
-func NewDummyDA() *DummyDA {
+func NewDummyDA(opts ...func(*DummyDA) *DummyDA) *DummyDA {
 	da := &DummyDA{
-		mu:   new(sync.Mutex),
-		data: make(map[uint64][]kvp),
+		mu:          new(sync.Mutex),
+		data:        make(map[uint64][]kvp),
+		maxBlobSize: DefaultMaxBytes,
```

Comment:
if `DefaultMaxBytes` is too high for dummyDA for testing, why even have that? just set some reasonable number which allows easier testing. i would still like to get rid of the celestia specific consts. also, if you could make a PR to celestia-da implementing this method, would be great.